reference/changelog.md

```diff
@@ -58,6 +63,103 @@ redirect_from: /install/changelog.html
+- Expose 'factory functions' from pulumi-aws. `[pulumi/pulumi-aws#304](https://github.com/pulumi/pulumi-aws/pull/304)`
+- Use stronger IAM doc types. `[pulumi/pulumi-aws#311](https://github.com/pulumi/pulumi-aws/pull/311)`
+- Auto-name aws.elasticache.Cluster#clusterId, `[pulumi/pulumi-aws#313](https://github.com/pulumi/pulumi-aws/pull/313)`
+
+### @pulumi/aws-infra 0.15.1
+
+- Update dependencies versions for `@pulumi/pulumi` and `@pulumi/aws`.
+
+### @pulumi/aws-serverless 0.15.1
+
+- Update dependencies versions for `@pulumi/pulumi` and `@pulumi/aws`.
+
+### @pulumi/azure 0.15.2
+
+- Consume newer terraform-provider from upstream. `[pulumi/pulumi-azure#93](https://github.com/pulumi/pulumi-azure/pull/93)`
+- Fix name limit for azure container groups. `[pulumi/pulumi-azure#94](https://github.com/pulumi/pulumi-azure/pull/94)`
+- Support provider configuration and property values provided by environment variaibles. `[pulumi/pulumi-azure#98](https://github.com/pulumi/pulumi-azure/pull/98)`
+
+### @pulumi/azure-serverless 0.15.2
+
+- Add support for hooking up FunctionApps to ServiceHub topics. `[pulumi/pulumi-azure-serverless#11](https://github.com/pulumi/pulumi-azure-serverless/pull/11)`
+- Allow include/exclude arguments to be passed from higher layers into FunctionApp creation. `[pulumi/pulumi-azure-serverless#15](https://github.com/pulumi/pulumi-azure-serverless/pull/15)`
+- Add Queue subscriptions (thanks to **[@mikhailshilkov](https://github.com/mikhailshilkov)**). `[pulumi/pulumi-azure-serverless#17](https://github.com/pulumi/pulumi-azure-serverless/pull/17)`
+
+### @pulumi/cloud, @pulumi/cloud-aws, @pulumi/cloud-azure 0.15.1
+
+As part of this release we are including a beta implementation of `@pulumi/cloud` on Azure.  You can now set the `cloud:provider` configuration value to `azure` to deploy applications writing against `@pulumi/cloud` on Azure.
+
+- Add a default lifecycle policy for ECR repos. `[pulumi/pulumicloud#556](https://github.com/pulumi/pulumi-cloud/pull/556)`
+- Add `cloud.HttpServer` making it easy to run express style apps across clouds. `[pulumi/pulumi-azure-cloud#562](https://github.com/pulumi/pulumi-cloud/pull/562)`
+- Log docker messages to the Task/Service we're creating, not the Repository resource we're using. `[pulumi/pulumi-azure-cloud#563](https://github.com/pulumi/pulumi-cloud/pull/563)`
+
+### @pulumi/docker
+
+- Pass logging messages to the right sink when doing docker operations. `[pulumi/pulumi-docker#7](https://github.com/pulumi/pulumi-docker/pull/7)`
+- Add `docker.Image` component. `[pulumi/pulumi-docker#10](https://github.com/pulumi/pulumi-docker/pull/10)` 
+
+### @pulumi/gcp 0.15.1
+
+- Support provider configuration and property values provided by environment variaibles. `[pulumi/pulumi-gcp#43](https://github.com/pulumi/pulumi-gcp/pull/43)`
+
+### @pulumi/kubernetes 0.17.0
+
+**TODO: Alex**
```

Comment:
- Incremental updates for Kubernetes resources will no longer be printed at the end of a `pulumi up`.
- More detailed explanations (and sometimes guidance) when an operation fails (_e.g._, "You cloud provider does not seem to support `Service`s` of type `LoadBalancer`).
- Fix several APIs, including allowing `ConfigFile` to take a file, and making `CollectionComponentResource.getResource` well-typed.
- Bug fixes in await logic, Windows support.

Reply:
I would lead with a top-level headline like "Cloud Native SDK for Kubernetes is available and recommended for production use."